packages/react-instantsearch/src/widgets/CurrentFilters/index.js

```diff
@@ -1,8 +1,25 @@
+/**
+ * @module widget/CurrentFilters
+ */
 import connect from './connect';
-import CurrentFilters from './CurrentFilters';
+import CurrentFiltersComponent from './CurrentFilters';
```

Comment:
Interesting point! Previously we were constructing the widgets in an anonymous way. In every file we had `Connected` as the variable name. Instead now, we are writing down the name of the widget / component as it is documented. So if you look in the code for Hits you will find matches in the index.js as well, the code is more explicit about the intention and, the @name is now inferred from the code.